App.svelte

```diff
@@ -48,14 +67,14 @@
     <button type="button" on:click={toggleInstructions}>Instructions</button>
   </div>
   <div slot="sidebar">
-    <h1>{authorityName}</h1>
+    <h1>{authorityName} <ZoomOutMap {boundaryGeojson} /></h1>
```

Comment:
The motivation behind switching from context to store for the MapLibre map is here. Context is based on the hierarchy of components -- it's only available to children of the place that sets it. So previously, only the 4 things underneath `Map` (`BoundaryLayer`, `DrawControls`, etc) could get the map. But... we have a button in the sidebar that needs to change the viewport.

I've been thinking through this problem earlier, and wrote up options at https://github.com/dabreegster/osm2svelte#map-and-sidebar-interactions. Option 3 is what we've followed in this codebase so far. We have stores like `currentSidebarHover` and `currentMapHover` that're global to the whole app, and sometimes stuff in the sidebar (not in the same part of the component tree as the Map) will change things here. But that feels like total overkill here -- I don't want to insert a little listener component somewhere under `Map` just to handle this button click. That'd be putting code to handle one piece of behavior in two different places.

I've also been looking through https://github.com/dimfeld/svelte-maplibre for inspiration how to handle some harder problems around only working with fully-loaded maps, which is a bit relevant here. They're using contexts, but also don't really have any examples with controls that live "outside" the `Map` subtree